serverless-plugin/index.js

```diff
@@ -299,15 +297,15 @@ class ServerlessPlugin {
 
             const res = this.runStack(directory, buildCommand);
 
-            // Copy the executable to the destination directory
+            // Copy the executable and runtime to the destination directory
```

Comment:
Where is the runtime copied - is this comment left around from some removed changes?

Reply:
Fixed, thanks! https://github.com/seek-oss/serverless-haskell/pull/101/commits/03b49e1bdeb260776752daf00ed7dcfc70bef6e7